apps/backend/src/modules/rating/typedefs/rating.ts

```diff
@@ -0,0 +1,123 @@
+import { gql } from "graphql-tag";
+
+const typedef = gql`
+enum MetricName {
+    "Numerical metrics"
+    Usefulness
+    Difficulty
+    Workload
+
+    "Boolean metrics"
+    Attendance
+    Recording
+}
+
+enum Semester {
```

Comment:
You shouldn't have to redefine this variable since `Semester` has already been defined in the `term` module.